# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/gpgme/hotfix-closefrom.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- gpgme-1.16.0/src/posix-io.c.vanilla	2021-10-27 14:11:35.607985639 +0200
+++ gpgme-1.16.0/src/posix-io.c	2021-10-27 14:12:10.086988098 +0200
@@ -570,7 +570,7 @@
               if (fd_list[i].fd > fd)
                 fd = fd_list[i].fd;
             fd++;
-#if defined(__sun) || defined(__FreeBSD__)
+#if 1
             closefrom (fd);
             max_fds = fd;
 #else /*!__sun */
